packages/chain-adapters/src/solana/SolanaChainAdapter.ts

```diff
@@ -203,14 +204,13 @@ export class ChainAdapter implements IChainAdapter<KnownChainIds.SolanaMainnet>
     }
   }
 
-  async buildSendTransaction(input: BuildSendTxInput<KnownChainIds.SolanaMainnet>): Promise<{
+  async buildSendApiTransaction(input: BuildSendApiTxInput<KnownChainIds.SolanaMainnet>): Promise<{
```

Comment:
Required for Solana in swapper - we don't have `wallet` to pass around there, and this method is what we use for all other chain families to build a Tx input without a wallet, just looks like it was ommitted for Solana?